README.md

```diff
@@ -61,6 +61,7 @@ Options:
   -t|--target             Path to the test runner application.
   -a|--targetargs         Arguments to be passed to the test runner.
   -o|--output             Output of the generated coverage report
+  -v|--verbosity          Sets the verbosity level of the command. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed].
```

Comment:
would you mind remove duplicate `--include-directory ...` switch explanation duplicate below?

Reply:
done